rali/rali_pybind/example/tf_pipeline.py

```diff
@@ -0,0 +1,73 @@
+
+from amd.rali.plugin.tf import RALIClassificationIterator
+from amd.rali.pipeline import Pipeline
+import amd.rali.ops as ops
+import amd.rali.types as types
+import sys
+import tensorflow as tf
+
+class HybridPipe(Pipeline):
+	def __init__(self, batch_size, num_threads, device_id, data_dir, crop, rali_cpu = True):
```

Comment:
Codacy found an issue: [Missing docstring in __init__ (D107)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5763058)